server/app/src/androidTest/java/androidx/test/uiautomator/UiScrollableCustom.java

```diff
@@ -1,10 +1,15 @@
 package androidx.test.uiautomator;
 
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
 public class UiScrollableCustom extends UiScrollable {
     public UiScrollableCustom(UiSelector container) {
         super(container);
     }
 
+    private static final Long TIMEOUT = 1000L;
```

Comment:
Waiting 1 second between scrolling attempts seems to be too much. Do you know the value for `etMaxSearchSwipes()` ? What's is the combined waiting time in case there is no element or if it requires all scrolling steps?

Reply:
Reducing less than 1 second is creating instability hence keeping 1 second is the best case. 
setMaxSearchSwipes() by default it is 30 but it is customised and overridden to include only 10. 
The combination of waiting time when the element is matching will be 10 seconds + scrolling time of around 0.5 seconds. which in Total 15 seconds.